spinflip/diagonalizer.py

```diff
@@ -49,22 +44,29 @@ def timescales(self, minmax = True):
         else:
             return hbar/differences
         
-    def state_evolution_plotter(self, t_lim = 'timescale', resolution=500, quantity = 'state_right', ylim = None, init_array = np.diag((1,0,0,0,0,0)), savefig = False):
+        #time array of neutrino density matrix, with init_array as intial condition
+    def state_evolution(self, resolution, t_lim, init_array):
```

Comment:
It would be good to leave the function in the same place and keep consistent whitespace so it doesn't look like a change. Fix the whitespace, but we can leave the function here.

Reply:
Can you clarify? not sure what I changed here

Reply:
I just meant that the function moved from line 52 to line 47. But it's fine to leave it here now.